framework/src/meshgenerators/MeshGenerator.C

```diff
@@ -21,6 +27,10 @@ MeshGenerator::validParams()
                         "Whether or not to show mesh info after generating the mesh "
                         "(bounding box, element types, sidesets, nodesets, subdomains, etc)");
 
+  params.addParam<bool>(
+      "output", false, "Whether or not to output the mesh file after generating the mesh");
+  params.addParam<bool>(
+      "nemesis", false, "Whether or not to output the mesh file in distributed form");
```

Comment:
```suggestion
      "nemesis", false, "Whether or not to output the mesh file in the nemesis format (only if output = true)");
```

You'll probably want to do this one on your computer, it'll probably fail clang format